README.md

```diff
@@ -143,6 +145,32 @@ https://github.com/domosekai
   releasing the build.
 - More details at https://www.softether.org/.
 
+# Comparison with Stable Edition
+
+| Protocol | Stable | Developer | Comment |
+| --- | --- | --- | --- |
+| SSL-VPN | ✅ | ✅ | |
+| OpenVPN | ✅ | ✅ | |
+| IPsec | ✅ | ✅ | |
+| L2TP | ✅ | ✅ | |
+| MS-SSTP | ✅ | ✅ | |
+| L2TPv3 | ✅ | ✅ | |
+| EtherIP | ✅ | ✅ | |
+| WireGuard | ❌ | ✅ ||
+| IKEv2 | ❌ | ❌ ||
+
+| Feature | Stable | Developer | Comment |
+| --- | --- | --- | --- |
+| Password Authentication | ✅ | ✅ | |

```

Comment:
I think RADIUS Authentication needs to be added.

Reply:
Added. Thanks.